x-pack/plugins/monitoring/public/lib/setup_mode.tsx

```diff
@@ -222,6 +225,7 @@ export const isInSetupMode = (context?: ISetupModeContext) => {
 };
 
 export const isSetupModeFeatureEnabled = (feature: SetupModeFeature) => {
+  if (isReactMigrationEnabled()) return setupModeReact.isSetupModeFeatureEnabled(feature);
```

Comment:
This method is used in many places in react child components rendered also in angular. This is a hack to make it work with the new file without modifying several components.